packages/authorization-server/jwt-authorization-grant.js

```diff
@@ -120,7 +120,7 @@ export default async (_, provider) => {
     console.log('Client requested scopes', ctx.oidc.params.scope);
 
     const client_requested_scopes = ctx.oidc.params.scope ? ctx.oidc.params.scope.split(' ') : [];
-    const idp_authorized_scopes = claims.scope.split(' ');
+    const idp_authorized_scopes = claims.scope ? claims.scope.split(' ') : "";
```

Comment:
nit: Can we use new syntax?

```suggestion
    const idp_authorized_scopes = claims.scope?.split(' ') ?? '';
```